tests/manager/manager-ctor-auto_encryption-error-001.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): incomplete auto encryption options
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/tools.php';
+
+$tests = [
+    [],
+    ['keyVaultNamespace' => 'admin.keys'],
+];
+
+foreach ($tests as $driverOptions) {
+    echo throws(function() use ($driverOptions) {
+        $manager = new MongoDB\Driver\Manager(null, [], ['autoEncryption' => $driverOptions]);
+    }, 'MongoDB\Driver\Exception\RuntimeException'), "\n\n";
+}
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+OK: Got MongoDB\Driver\Exception\RuntimeException
+Key vault namespace option required
```

Comment:
I assume these error messages originate from libmongocrypt or libmongoc, as I don't see them in this PR. It's unfortunate that they don't actually tell users what option key is missing, but I suppose there's no way around that. I'm not going to propose that we parse out these error messages and rewrite them :stuck_out_tongue:

Reply:
Yes, those are errors from libmongoc. If we want to make those error messages more meaningful, we need to start checking error codes or something, which is a bigger undertaking.